testing.rst

```diff
@@ -163,7 +163,7 @@ As an example, a test could look like this::
 
             $client->request('GET', '/post/hello-world');
 
-            $this->assertEquals(200, $client->getResponse()->getStatusCode());
+            $this->assertResponseStatusCodeSame(200);
```

Comment:
I would use `assertResponseIsSuccessful()` instead.

Reply:
`assertResponseIsSuccessful()` only checks for `2xx` status code, not only `200`